docs/annotations.md

```diff
@@ -183,3 +184,37 @@ spec:
           serviceName: go-server-service
           servicePort: 80
 ```
+
+## Use Private IP
+
+This annotation allows to specify whether to expose this endpoint on Private IP of Application Gateway.
+
+> **Note**
+1) Application Gateway doesn't support multiple IPs on the same port (example: 80/443). So, if you have one ingress with annotation `use-private-ip: "false"` and other with `use-private-ip: "true"` on `HTTP`, they will fail the update on the Gateway.
```

Comment:
I'd remove `So, if you have one ` and leave it at `Ingress with annotation`
also I'd change `other` with `another`.
I'd change `they will` with `AGIC will fail to update...`